Remotion/ObjectBinding/Web/UI/Controls/Validation/BocListValidationResultDispatchingValidator.cs

```diff
@@ -102,8 +121,24 @@ private void DispatchToValidationFailureRepository (IBusinessObjectValidationRes
 
     protected override bool EvaluateIsValid ()
     {
-      // TODO RM-6056: Shows a validation error if IBusinessObjectValidationResult returned messages for this control.
-      return true;
+      var sb = new StringBuilder();
+
+      foreach (var validationFailure in _validationFailuresOfList)
+      {
+        sb.AppendLine();
+        sb.Append(validationFailure.ErrorMessage);
+      }
+
+      if (_numberOfNotVisibleValidationErrors > 0)
+      {
+        var remainingValidationFailureText = ((IControlWithResourceManager)GetControlToValidate()).GetResourceManager().GetString(BocList.ResourceIdentifier.RemainingValidationFailureText);
+        sb.AppendLine();
+        sb.AppendFormat(remainingValidationFailureText, _numberOfNotVisibleValidationErrors);
```

Comment:
Please check, I think you need to put the AppendLine at the end of the text line, not the beginning.